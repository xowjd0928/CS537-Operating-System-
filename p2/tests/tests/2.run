cd ../solution; ../tests/run-xv6-command.exp CPUS=1 Makefile.test test_2 | grep -E 'XV6_TEST_OUTPUT|XV6_TEST_ERROR'; cd ../tests