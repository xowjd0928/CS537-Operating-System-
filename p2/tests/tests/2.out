XV6_TEST_OUTPUT Null pointers handled correctly.
